src/store/post-store.js

```diff
@@ -60,5 +60,61 @@ export const postStore = (() => {
             console.log(error)
         }
     }
-    return { addPost, getAllPosts };
+    const addCommentOnPost = async (commentText, postId) => {
+        try {
+            const commentTitle = commentText;
```

Comment:
do you really need this const commentTitle = commentText;?
you can directly change variable name in argument declaration @ShivanshSinghnnovative